client/src/app/shared/components/uuid/uuid.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="uuid-display">
+    <p (click)="copyUuid()">ID: {{uuid}}</p>
```

Comment:
it might be good to put a note that clicking will copy to the clipboard. Or yeah, could just wait for Gabs/wrangler's input.

Reply:
Yeah gabs will decide